README.md

```diff
@@ -463,4 +463,47 @@ def ==(other)
   other.is_a?(Quantity) && ...
   # ...or, sometimes, duck type
   other.respond_to?(:amount) && other.respond_to?(:unit) && ...
+end
+```
+
+## Pattern Matching
+
+Value objects are a prime candidate for use with the pattern matching syntax introduced in Ruby 2.7.
+
+The easiest way to integrate with pattern matching is to return a hash of attributes from `#decontruct_keys`, which you have probably already implemented in `#to_h`. This allows the object to be pattern matched using the hash syntax:
```

Comment:
I am far from being native speaker, but it seems some rephrasing is necessary here, currently, it reads "...`#deconstruct_keys`, which you have probably already implemented...". Something like (see also comment below):
> ...is to return a hash of attributes from `#decontruct_keys`. The most simple (and often enough) implementation is to just call `#to_h`, if you have it already implemented.

Or something like that.

Reply:
Yep, I've worded this a bit better now.